_data/partners.yaml

```diff
@@ -120,6 +120,8 @@
     The interns have the opportunity to work on a variety of cloud native technologies. Based on the area of their individual interest they can choose the technologies they want to learn or sharpen their skills.
     
     To see the participating interns please checkout <a href="https://meshery.io/community/">our community members</a> page.
+    <br>
+    <a href="https://uem.edu.in/blog/what-is-open-source/"> Learn More </a>
```

Comment:
Thanks, @ashiscs. Will you update the existing Learn more about our Partner hyperlink instead of creating a new one?